README.md

```diff
@@ -10,7 +10,7 @@
 Inference of [LLaMA](https://arxiv.org/abs/2302.13971) model in pure C/C++
 
 ### Hot topics
-
+- ‼️ BPE tokenizer update: existing Falcon 180B `.gguf` models will need to be reconverted
```

Comment:
The update actually breaks all Falcon and Starcoder models, not just Falcon 180B.